### PR TITLE
OSD-22496: add scc workaround

### PR DESCRIPTION
**Pull request by typeid**
Adds workaround for pipelines-scc missing the seccompProfile: https://issues.redhat.com/browse/SRVKP-4372

### CODE REVIEW

**Comment by typeid** [comment]
/hold

Will get review from app-sre as well.

**Comment by RaphaelBut** [comment]
looks good to me as well

File: openshift/SRVKP-437-workaround-template.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              command: ["/bin/sh", "-c"]
+              args:
+              - |
+                scc_name="pipelines-scc"
+
+                # Current namespace as script can be deployed to different namespaces (prod/stage namespaces differ)
+                namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
+
+                echo "Reapplying runtime/default: $seccomp_profile to $scc_name in namespace $namespace"
+                oc patch scc $scc_name -n $namespace --type=json -p '[{"op": "replace", "path": "/seccompProfiles", "value": ["runtime/default"]}]'
```
[the remark this answers is not in the document]
Comment (typeid) [reply openshift/SRVKP-437-workaround-template.yaml]: Thanks, I have added a check to make sure we only patch the scc if the `seccompProfiles` is empty.

**Pull request merged**